.travis/script.sh

```diff
@@ -3,7 +3,7 @@ set -x
 
 # If this is a master build then deploy at the end
 if [ "$TRAVIS_BRANCH" = 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
-  mvn -Dstyle.color=always -B -U -P sign --settings travis-settings.xml deploy;
```

Comment:
You don't need to add verify here, deploy is after verify in the maven build lifecycle